scripts/background.js

```diff
@@ -25,12 +25,15 @@ var screenShotURL;
 		// toggle the visibility of slideout (I didn't test this when the slider is open yet, beware)
 		chrome.tabs.executeScript({ code: `(${ inContent })()` });
 		function inContent() {
-  			const el = document.getElementById('slideout');
+			  const el = document.getElementById('slideout');
```

Comment:
Can we make these vars more descriptive? 😅  Maybe slideOutEl and genderMagFrameEl?